.goreleaser.yml

```diff
@@ -0,0 +1,57 @@
+---
+project_name: gaia
+
+release:
+  github:
+    owner: marbar3778
+    name: gaia
+
+builds:
+  - binary: gaiad
+    goos:
+      - darwin
+      - windows
+      - linux
+    goarch:
+      - amd64
+      - arm64
+      - arm
+    goarm:
+      - 6
+      - 7
+    main: ./cmd/gaiad/
+    flags:
+      - -tags=netgo,ledger
+    ldflags: -s -w - trimpath -X github.com/cosmos/cosmos-sdk/version.Name=gaia -X github.com/cosmos/cosmos-sdk/version.ServerName=gaiad -X github.com/cosmos/cosmos-sdk/version.ClientName=gaiacli -X github.com/cosmos/cosmos-sdk/version.Version={{ .Version }} -X github.com/cosmos/cosmos-sdk/version.Commit={{ .Commit }} -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger"
```

Comment:
You should add `-extldflags=-static -buildid=${VERSION}`. Mind testing that too please?

Reply:
this was removed, when i get back to testing this i will try it